src/main/java/ip/task/Task.java

```diff
@@ -0,0 +1,42 @@
+package ip.task;
+
+public class Task {
+
+    protected String description;
+    protected boolean isDone;
+
+    /**
+     * Constructor
+     *
+     * @param description Description of task.
+     */
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    public String getDescription() {
+        return description;
+    }
+
+    public void markAsDone() {
+        isDone = true;
+    }
+
+    /**
+     * Returns a status icon depending on whether task isDone.
+     * @return Tick symbol if isDone else X symbol.
+     */
```

Comment:
Consider leaving an empty line in between the description and parameter for this Javadoc comments